tox.ini

```diff
@@ -59,7 +59,7 @@ commands_pre =
 
 commands =
     !cov: pytest --pyargs quantity {posargs} {toxinidir}/tests {toxinidir}/docs
-    cov: pytest --pyargs quantity --cov quantity --cov-report xml:coverage.xml --cov-report term-missing {posargs} {toxinidir}/tests {toxinidir}/docs
+    cov: pytest --pyargs quantity --cov src --cov-report xml:coverage.xml --cov-report term-missing {posargs} {toxinidir}/tests {toxinidir}/docs
```

Comment:
This one also should be just `quantity` (tested locally; we don't have a coverage test run on CI yet). I.e., `tox.ini` can remain unchanged.